{{cookiecutter.project_name}}/src/{{cookiecutter.package_name}}/pp/basic.py

```diff
@@ -15,3 +24,68 @@ def basic_preproc(adata: AnnData) -> int:
     """
     print("Implement a preprocessing function here.")
     return 0
+
+
+def elaborate_example(
+    items: Iterable[ScverseDataStructures],
+    transform: Callable[[Any], str],
+    /,  # indicates the end of positional only arguments
```

Comment:
There are very few cases where positional-only arguments make sense. Not sure I'd show this here. 
```suggestion

```